Tasks/Common/securefiles-common/package.json

```diff
@@ -18,6 +18,6 @@
   "homepage": "https://github.com/Microsoft/vsts-tasks#readme",
   "dependencies": {
     "vsts-task-lib": "^2.0.3-preview",
-    "vso-node-api": "^6.1.2-preview"
+    "vso-node-api": "^6.5"
```

Comment:
A breaking change was introduced in 6.3-preview. (`getTaskAgentApi()` now returns a Promise.) Moving to 6.5 so we'll consistently get the new behavior.

Reply:
@bryanmacfarlane @stephenmichaelf and I are going to discuss why it started using 6.5.